HHS/OPRE-OPS
# feat: ✨ (4) 2349:  Approve Agreement toggle action

## What changed

- adds BLIDiffTable and friends
- adds Review Documents Accordion for **Executing Status Changes**
- adds styles on BLIDifRow
- toggle actions for current/approved state in BLIDiffTable and Summary Cards

Does Not Include
- Can Review Accordion toggle actions which will be part of round 5

## Issue

#2349 

## How to test

1. Create a change requests by selecting Agreement 9
2. edit a BLI for either a budget or status change
3. goto review tab
4. select the change request
5. see BLIDiffTable which highlights changed budget line properties
7. see Summary Cards 
8. toggle before/after approval
9. see changes
10. 💰 profit

Or 

1. see automated tests pass

## Screenshots

https://github.com/user-attachments/assets/1dc6f480-42a9-4bfc-850e-9d8142155267

## Definition of Done Checklist

## Links

Links to prior PRs:
1. https://github.com/HHS/OPRE-OPS/pull/2516
11. https://github.com/HHS/OPRE-OPS/pull/2544
12. https://github.com/HHS/OPRE-OPS/pull/2554
13. https://github.com/HHS/OPRE-OPS/pull/2558
14. https://github.com/HHS/OPRE-OPS/pull/2570
